sklearn/ensemble/tests/test_forest.py

```diff
@@ -163,6 +176,30 @@ def test_iris(name, criterion):
     check_iris_criterion(name, criterion)
 
 
+def check_imbalanced_criterion(name, criterion):
+    ForestClassifier = FOREST_CLASSIFIERS[name]
+
+    clf = ForestClassifier(n_estimators=10, criterion=criterion, random_state=1)
+    clf.fit(X_large_imbl, y_large_imbl)
+
+    # score is a mean of minority class predict_proba
+    score = clf.predict_proba(X_large_imbl)[:, 1].mean()
+
+    assert (
+        score > imbl_minority_class_ratio
+    ), "Failed with imbalanced criterion %s, score = %f, minority class ratio = %f" % (
+        criterion,
+        score,
+        imbl_minority_class_ratio,
+    )
+
+
+@pytest.mark.parametrize("name", FOREST_CLASSIFIERS)
+@pytest.mark.parametrize("criterion", ["hellinger"])
+def test_imbalanced_criterions(name, criterion):
+    check_imbalanced_criterion(name, criterion)
+
```

Comment:
@jjerphan, @glemaitre Do you know why we use this test pattern (one function with the logic and another parametrized that calls it)?
`sklearn/ensemble/tests/test_forest.py` has some tests like this.

Reply:
I do not know and I would merge them together.